java-buildpack/bin/build

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+echo "---> Java buildpack"
+
+set -eo pipefail
+
+env_dir=$1/env
+cache_dir=$2
+launch_dir=$3
+
+# Load user-provided build-time environment variables
+if compgen -G "$env_dir/*" > /dev/null; then
+  for var in "$env_dir"/*; do
+    declare "$(basename "$var")=$(<"$var")"
+  done
+fi
+
+echo "---> Installing JDK"
+
+if [[ ! -f $launch_dir/jdk.toml ]]; then
+  mkdir -p $launch_dir/jdk
+  jdk_url="https://cdn.azul.com/zulu/bin/zulu8.28.0.1-jdk8.0.163-linux_x64.tar.gz"
+  curl -sfL "$jdk_url" | tar pxz -C $launch_dir/jdk --strip-components=1
+  mkdir -p $launch_dir/jdk/.profile.d
```

Comment:
@jkutner this should be `profile.d` instead of `.profile.d`

Reply:
Could we define the openjdk version to be installed such as the distro ?

Reply:
@cmoulliard yea, absolutely. I only grabbed Zulu 8u163 to get something working.